components/automate-ui/src/app/helpers/auth/sorter.ts

```diff
@@ -0,0 +1,29 @@
+
+export class ChefSorters {
+
```

Comment:
Named the class ChefSorters incase we may want to add different types of sorting functions in the future